.ansible-lint

```diff
@@ -0,0 +1,19 @@
+profile: production
+exclude_paths:
+    - mkdocs.yml
+    - docs/
+    - tests/config/
+    - tests/secrets.sample.yaml
+    - tests/vars.sample.yaml
+skip_list:
+    - var-naming[no-role-prefix]
+    - yaml[line-length]
+    - risky-shell-pipe # we have pipefail set in common_defaults shell_header:
+    - name[casing] # do we want that one for consistency? maybe?
+    - no-changed-when # any point adding changed_when: false in all our shell?
+    - command-instead-of-shell
+    - latest[git] # yes we want to clone latest of install_yamls
```

Comment:
for this one since there was only one hit (install_yamls) we might consider skipping just that one task/locally instead of this global skip_list.

Reply:
k done update incoming